tutorials/28_Structured_Output_With_Loop.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
<p>A few comments here: </p><ol><li>It seems like 'OutputParser' is actually an output validator? Why is the name output parser here? </li><li>It's unclear here why there are 'invalid replies' created in the run function. An explanation on what's happening here would be useful</li><li>The instructions on what this code cell is meant for can be clearer. For example when a user sees the commented out section inside the code block, they might think this is meant for them to edit. However it seems you're simply <em>providing</em> a custom component? </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>